locales/en/LC_MESSAGES/volto.po

```diff
@@ -3974,12 +3974,12 @@ msgstr "Event type"
 #: components/ItaliaTheme/View/PersonaView/PersonaRuolo
 # defaultMessage: Tipo di incarico
 msgid "tipologia_incarico"
-msgstr ""
+msgstr "Type of assignment"
```

Comment:
"Assignment type"

Anche sotto hai usato "Venue type" e non "Type of venue" :) suona più naturale no?